### DISCUSSION TITLE
wiki written in vlang powered by redis

### DISCUSSION THREAD

**Discussion by hiqsociety** [vlang/v#14608] [Ideas]
wiki written in vlang backed by redis... that'll be something

**Comment by JalonSolov** [comment]
Wiki written in V, powered by vsql.  More portable to every system V supports.

**Comment by dumblob** [comment]
[Offline-first](https://www.inkandswitch.com/local-first.html ) wiki (i.e. with a visual intuitive diff - maybe additionally accompanied with [diamond types - i.e. submillisecond-fast CRDTs](https://github.com/josephg/diamond-types )) would be even better :wink:.
